package.json

```diff
@@ -12,15 +12,24 @@
   "bugs": {
```

Comment:
Seems like we should publish this under an explicit namespace for our npm org, maybe we could name it `@recordreplay/protocol` or something?